### ISSUE TITLE
学习率 无效

### ISSUE THREAD

**Issue by WangJ970316** [PaddlePaddle/Paddle#29742]
模型反向传播梯度在更新，但是不受学习率的控制。就是 不管怎么设置lr，对模型的梯度都没有影响。每次参数更新的方向和幅度都是固定的。更换优化器也没有效果

- 版本、环境信息：
    1）PaddlePaddle版本：1.8.3
    3）GPU：请提供GPU型号，CUDA 10.1.168 CUDNN 7.6.0
    4）Python 3.7
 注：您可以通过执行[summary_env.py](https://github.com/PaddlePaddle/Paddle/blob/develop/tools/summary_env.py)获取以上信息。
- 模型信息
    1）BILSTM
def bilstm_net(data,
               seq_len,
               dict_dim,
               emb_dim=128,
               hid_dim=128,
               hid_dim2=96,
               emb_lr=30.0,
               is_prediction=False):
    """
    Bi-Lstm net
    """
    # embedding layer
    emb = fluid.layers.embedding(
        input=data,
        size=[dict_dim, emb_dim],
        param_attr=fluid.ParamAttr(learning_rate=emb_lr))

    flow_seq_len = fluid.layers.reshape(x=seq_len,shape=[-1])
    emb = fluid.layers.sequence_unpad(emb, length=flow_seq_len)
    fc0 = fluid.layers.fc(input=emb, size=hid_dim * 4)
    rfc0 = fluid.layers.fc(input=emb, size=hid_dim * 4)
    lstm_h, c = fluid.layers.dynamic_lstm(
        input=fc0, size=hid_dim * 4, is_reverse=False)
    rlstm_h, c = fluid.layers.dynamic_lstm(
        input=rfc0, size=hid_dim * 4, is_reverse=True)
    lstm_concat = fluid.layers.concat(input=[lstm_h, rlstm_h], axis=-1)
    fluid.layers.Print(lstm_concat, message="lstm_concat")
    sequence_out = fluid.layers.fc(input=lstm_concat, size=1)
    lstm_last = fluid.layers.sequence_last_step(input=lstm_h)
    rlstm_last = fluid.layers.sequence_last_step(input=rlstm_h)
    # concat layer
    lstm_concat_pol = fluid.layers.concat(input=[lstm_last, rlstm_last], axis=1)
    pool_out = fluid.layers.fc(input=lstm_concat_pol, size=1)
    return sequence_out, pool_out, lstm_concat, lstm_concat_pol, emb
